pkg/config/organization.go

```diff
@@ -0,0 +1,175 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package config
+
+import (
+	"fmt"
+
+	"github.com/golang/protobuf/proto"
+	cb "github.com/hyperledger/fabric-protos-go/common"
+	mb "github.com/hyperledger/fabric-protos-go/msp"
+	pb "github.com/hyperledger/fabric-protos-go/peer"
+)
+
+// GetApplicationOrg retrieves a existing org from an application organization config group.
+func GetApplicationOrg(config cb.Config, orgName string) (Organization, error) {
+	orgGroup, ok := config.ChannelGroup.Groups[ApplicationGroupKey].Groups[orgName]
+	if !ok {
+		return Organization{}, fmt.Errorf("application org %s does not exist in channel config", orgName)
+	}
+
+	return getOrganization(orgGroup, orgName)
+}
+
+// GetOrdererOrg retrieves a existing org from an orderer organization config group.
+func GetOrdererOrg(config cb.Config, orgName string) (Organization, error) {
+	orgGroup, ok := config.ChannelGroup.Groups[OrdererGroupKey].Groups[orgName]
+	if !ok {
+		return Organization{}, fmt.Errorf("orderer org %s does not exist in channel config", orgName)
+	}
+
+	org, err := getOrganization(orgGroup, orgName)
+	if err != nil {
+		return Organization{}, err
+	}
+
+	// Orderer organization requires orderer endpoints.
+	endPointsProtos := &cb.OrdererAddresses{}
+	err = unmarshalConfigValueAtKey(orgGroup, EndpointsKey, endPointsProtos)
+	if err != nil {
+		return Organization{}, err
+	}
+	ordererEndpoints := []string{}
```

Comment:
Nit: since we already know the length of the endpoints, shouldn't we initialize this via `make([]string, len(endPointsProtos.Addresses))`?

Reply:
done